src/tests/infusion/createInfusion.test.js

```diff
@@ -43,6 +42,50 @@ const testCases = [
       },
     },
   },
+  {
+    title: 'creation of infusion should fail for a nurse user',
+    request: context => ({
+      path: '/api/infusion',
+      method: 'post',
+      body: infusion,
+      headers: {
+        'req-token': context.testGlobals[NURSE_USER].authToken,
+      },
+    }),
+    response: {
+      status: 403,
+      body: {
+        success: false,
+        message: 'You do not have access to this endpoint',
+      },
+    },
+  },
+  {
+    title: 'creation of infusion should fail if some of the required data such as patientName is missing',
```

Comment:
This test case is too broad, the fact that we get a correct validation response when `patientName` is missing does not guarantee that the same thing would happen when `startVolume` is missing so please add test case for all the possible validation.

Reply:
okay sir

Reply:
Fixed